apps/dav/tests/unit/CardDAV/AddressBookImplTest.php

```diff
@@ -355,8 +358,88 @@ public function testVCard2Array() {
 			],
 
 			'X-SOCIALPROFILE' => [
-				'twitter'=> 'tw-example',
-				'facebook'=> 'fb-example',
+				'tw-example',
+				'tw-example-2',
+				'fb-example',
```

Comment:
```suggestion
				['type' => 'twitter', 'value' => 'tw-example'],
				['type' => 'twitter', 'value' => 'tw-example-2'],
				['type' => 'facebook', 'value' => 'fb-example'],
```

Should be it from reading the code

Reply:
Nope, only if you have specified the withType, that's why I added the seconds tests

Reply:
Ah, now I see it